.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -19,3 +19,15 @@ One of the following will happen to feature requests:
     see the [contributor guide](https://github.com/seqan/seqan3/pull/CONTRIBUTING.md).
   * The feature request is invalid and will be closed.
 -->
+
+### Done
+
+- [ ] Implementation and design approved
+- [ ] Unit tests pass
+- [ ] Test coverage > 97%
```

Comment:
Why only 97?
Since we can exclude "unreachable" code (e.g. malloc throws), there isn't really a reason we shouldn't strive for 100%?

Reply:
Yes, we could have 100%, but not in all cases. There are special cases with the SIMD tests and SIMD alignment tests which cannot reach 100% as long as vectorised test infrastructure is not incorporated. Thus, marking the code unreachable is wrong and error prone since it is easy to forget that the things need to be removed once the infrastructure is added. As an alternative we could make 100% and only in the exceptional case where we know we can reach 100% but not yet, we allow a lower threshold. I just want to avoid as many exceptions as possible since it is important to adhere to the DoD.

Reply:
Well, then let's do 100% and allow a lower threshold for SIMD...